singleuser/user-fixes.py

```diff
@@ -0,0 +1,12 @@
+import os
+
+custom_path = os.path.expanduser('~/user-fixes.py')
+if os.path.exists(custom_path):
+    with open(custom_path, 'r') as f:
```

Comment:
The mode string can also include the letter 'b' either as a last
       character or as a character between the characters in any of the two-
       character strings described above.  This is strictly for
       compatibility with C89 and has no effect; the 'b' is ignored on all
       POSIX conforming systems, including Linux. 

from http://man7.org/linux/man-pages/man3/fopen.3.html.